### PR TITLE
Day 9 Edits pt. 1

### PR DESCRIPTION
**Pull request by Parzivus**
Small-ish update since I didn't want to sit on these edits for too long. Second half of day 9 will probably be finished by Thursday or Friday.

### CODE REVIEW

File: script/Arcueid/Day 9/09_00_ARC09_1.txt [hunk 1]
```diff
@@ -41,7 +41,8 @@ A landscape blanketed in a shimmering haze,
 [sha:3c325a238b489ee153bab2a33827624f7f5b2062]{
 -- Page 2, Offset 21300.
 -- 気が遠くなるような蝉の声。
-and the overwhelming cries of the cicadas.
+// Thoughts?
+and the overwhelming cries of cicadas.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_1.txt:45]: Minor pref for with `the`

File: script/Arcueid/Day 9/09_00_ARC09_1.txt [hunk 2]
```diff
@@ -91,7 +92,7 @@ A hot day in the middle of summer.
 [sha:2bf9438922df5ebec15dfa1911df8f13c532d541]{
 -- Page 4, Offset 21310.
 -- まるで、セカイがフライパンになったみたい。
-As if the world had turned into a frying pan.
+as if the world had turned into a frying pan.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_1.txt:95]: Revert? Prev sentence ends in `.`

File: script/Arcueid/Day 9/09_00_ARC09_1.txt [hunk 3]
```diff
@@ -136,8 +137,9 @@ A child lies crumpled in the clearing.
 [sha:9b13c7cf5b90dbcd5fdd4990fe9f41bdac9d2939]{
 -- Page 5, Offset 21319.
 -- 白いシャツが真っ赤に染まって、ぴくりとも動かない。
-A deep red stain has blossomed on its white shirt.
- It doesn't move so much as an eyebrow.
+// Any reason for "it?" "They" seems more appropriate for a human being.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_1.txt:140]: There are a couple places in the text where the corpse is referred to in a very inanimate fashion, but this isn't really one of them. This sentence doesn't actually have a direct reference to the child, so we can go with they/them.

File: script/Arcueid/Day 9/09_00_ARC09_1_1.txt [hunk 4]
```diff
@@ -55,6 +56,6 @@ The one thing that's bothering me is that reaction.
 [sha:1cf6cb3cadf66fbc02cfe87050dd25ecbad57f13]{
 -- Page 17, Offset 21359.
 -- 傷が治りきっていないのに暴れた反動だろう。今夜あたり、ちゃんと完治しているか確かめてやる。
-I'm sure she's just paying the price for being so rambunctious even though her wounds haven't healed yet.
- When I get the chance tonight I'll try and confirm if she really has fully recovered or not.
+I'm sure she's just paying the price for being so rambunctious, even though her wounds haven't healed yet.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_1_1.txt:59]: Not sure I agree with the comma here, to me it partitions the sentence such that "even though her wounds haven't healed yet" sounds like the state of _not_ being healed would make her _not_ pay the price.

File: script/Arcueid/Day 9/09_00_ARC09_2B.txt [hunk 5]
```diff
@@ -96,8 +97,8 @@ In the end, Hisui never actually touched my body,
 -- その後は琥珀さんの手をかりてベッドに戻り、病人食としてお粥をご馳走になり、今に至る。
 // 病人食 modulated a bit, not sure we have an exact match in EN.
 After that, I returned to the bed with Kohaku-san's assistance,
- was treated to some easy to digest rice porridge,
- and now here we are.
+ was treated to some easy-to-digest rice porridge,
+ and now, here we are.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_2B.txt:101]: Not sure I agree with this comma

File: script/Arcueid/Day 9/09_00_ARC09_2B.txt [hunk 6]
```diff
@@ -136,12 +137,13 @@ However, I strongly suggest that you receive a medical examination.
 -- Page 45, Offset 21431.
 -- 今朝は阿良句先生がいらっしゃいますので、これを機にしっかりお体を診ていただくのはいかがでしょうか？」
 Arach-sensei happens to be present this morning,
- perhaps you would like to take advantage of this opportunity to receive a thorough physical examination?"
+ so perhaps you would like to take advantage of this opportunity to receive a thorough physical examination?"
```
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_2B.txt:140]: Mild personal pref for no `so`

File: script/Arcueid/Day 9/09_00_ARC09_2_1.txt [hunk 7]
```diff
@@ -39,7 +39,7 @@ There's also a chance I suffered some internal bleeding as a result of my scuffl
 // Reading お客さん more as a joking honorific than as literal?
 My dear man, there doesn't appear to be anything wrong with you?
  I rushed over here expecting some excitement, like a blown off arm or a gouged out eye,
- so I have to say this has left me feeling a little frustrated.
+ so I have to say, this has left me feeling a little frustrated.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_2_1.txt:42]: Feels like a pause too many to me

File: script/Arcueid/Day 9/09_00_ARC09_2_1.txt [hunk 8]
```diff
@@ -246,6 +246,7 @@ Combined with her rather revealing garments,
 [sha:88b7075b3b32febcfa325d577e5a7a6679735120]{
 -- Page 65, Offset 21476.
 -- ……ふーん。ほーう。なるほどぅ……なんか使ったにせよ、完全に塞がってるわー。貧弱なのに代謝機能は人並みというか、生き残りやすいというかぁ。あーあー、すっっっごく残念。志貴チャン、何も引くものがないんだもの。これじゃあアタシも足しようがないって感じぃ？」
+// I don't know what to make of "I'm not sure that they used", can a JP speaker check this?
```
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_2_1.txt:249]: Oops, yeah this should be "I'm not sure **what** they used"

File: script/Ciel/Day 11/11_01_CIEL11_1.txt [hunk 9]
```diff
@@ -11,12 +11,13 @@ A blue sky with huge, towering cumulonimbus clouds.
 [sha:c12e997c9c8593402cca860e0f4c8623cb0ab971]{
 -- Page 1, Offset 28973.
 -- じりじりとゆらぐ風景と
-The landscape shimmers in the heat
+-- TRANSLATION HERE
```
Comment (rschlaikjer) [inline script/Ciel/Day 11/11_01_CIEL11_1.txt:14]: Seems like an error, but we can solve that later. Assuming the rest of the edits are reexports.

File: script/Arcueid/Day 9/09_00_ARC09_1_1.txt (hunk 4 above)
Comment (Avuxo) [reply script/Arcueid/Day 9/09_00_ARC09_1_1.txt:59]: Right. "rambunctious + not having wounds healed" is what she's paying the price for. Not "(paying price for being rambunctious) + wounds havent healed"

File: script/Arcueid/Day 9/09_00_ARC09_2.txt [hunk 10]
```diff
@@ -143,6 +143,7 @@ And so, I simply lay where I fell.
 [sha:122205647ac94447992b49f46e1dc73c282ac7b7]{
 -- Page 34, Offset 21407.
 -- 俺は床に倒れたまま、翡翠がもう一度<訪|おとず>れる８時過ぎまで、ぼんやりと時計の針を見上げていた。
+// I think the tense here is okay, but it could probably be switched to present tense if it matters much.
```
Comment (Avuxo) [inline script/Arcueid/Day 9/09_00_ARC09_2.txt:146]: jp is past anyways

File: script/Arcueid/Day 9/09_00_ARC09_2B.txt [hunk 11]
```diff
@@ -13,8 +13,9 @@ I did not notice that anything was amiss...
 [sha:5222c64e28982a05b1dd1f60fa99513d7ff7417d]{
 -- Page 10005, Offset 21410.
 -- 「翡翠は悪くないよ。いつもの事だし、そう大げさにとらない、とらない」
+// Three clauses in one sentence either needs to be split like this or use a semicolon
```
Comment (Avuxo) [inline script/Arcueid/Day 9/09_00_ARC09_2B.txt:16]: minor nit, but don't leave comments like this in script files. Without the context of the github pr diff, it doesn't really mean anything. Please remove before you merge

File: script/Arcueid/Day 9/09_00_ARC09_2B.txt (hunk 5 above)
Comment (Avuxo) [reply script/Arcueid/Day 9/09_00_ARC09_2B.txt:101]: I agree with it

File: script/Arcueid/Day 9/09_00_ARC09_2_1.txt [hunk 12]
```diff
@@ -6,7 +6,7 @@
 [sha:97b31b06a729c4c6e322785a4f6f5412d547b2ef]{
 -- Page 10006, Offset 21436.
 -- 琥珀さんは使用人の職務で忙しいだろうし、ここは阿良句博士にお願いしよう。
-Kohaku-san is probably busy enough as it is with her duties as a maid,
+Kohaku-san is probably busy enough as is with her duties as a maid.
```
Comment (Avuxo) [inline script/Arcueid/Day 9/09_00_ARC09_2_1.txt:9]: revert

**Review by Avuxo — APPROVED** [review]
lgtm pending comments and removal of reexports

File: script/Ciel/Day 11/11_01_CIEL11_1.txt (hunk 9 above)
Comment (Parzivus) [reply script/Ciel/Day 11/11_01_CIEL11_1.txt:14]: That's correct, my folder was missing some of the Ciel files. Shouldn't show up in future edits

**Comment by Parzivus** [comment]
Response to suggestions

**Comment by Parzivus** [comment]
I'll merge this in a few hours

**Pull request merged**